systemtest/src/test/java/io/strimzi/systemtest/CustomResourceStatusST.java

```diff
@@ -170,40 +173,64 @@ void testKafkaBridgeStatus() {
     @Test
     void testKafkaConnectStatus() {
         String connectUrl = "http://my-cluster-connect-api.status-cluster-test.svc:8083";
-        KafkaConnectResource.kafkaConnect(CLUSTER_NAME, 1).done();
+        KafkaConnectResource.kafkaConnect(CLUSTER_NAME, 1)
+            .editMetadata()
+                .addToAnnotations("strimzi.io/use-connector-resources", "true")
+            .endMetadata().done();
         waitForKafkaConnectStatus("Ready");
+        KafkaConnectorResource.kafkaConnector(CLUSTER_NAME, TOPIC_NAME).done();
+        waitForKafkaConnectorStatus(CLUSTER_NAME, "Ready");
         assertKafkaConnectStatus(1, connectUrl);
 
         KafkaConnectResource.replaceKafkaConnectResource(CLUSTER_NAME, kb -> kb.getSpec().setResources(new ResourceRequirementsBuilder()
                 .addToRequests("cpu", new Quantity("100000000m"))
                 .build()));
         waitForKafkaConnectStatus("NotReady");
 
+        KafkaConnectorResource.replaceKafkaConnectorResource(CLUSTER_NAME,
```

Comment:
I'm not sure I understand the logic why this is here. It doesn't seem to belong to this cluster and the actual Connect cluster is anyway not ready. I think the test with invalid lbel should be done when the Connect is ready.

Reply:
Connector itself doesn't reflect status of Kafka Connect so I think it doesn't matter where this code is. I can move to test phase, where connect is ready with same results I think. Or I can create completely new test case, but I wanted to save some time during execution.

Reply:
Why does it not reflect it? I do not think there is any guarantee that the connector can be `Ready` if the Connect is not `Ready`.

I understand why saving the time might be nice. But you should do this test when the Connect is `Ready`. That does not have to be different test case, or? You can just move it few lines up or down.

Reply:
It's better question for @tombentley. After our last discussion I understood that Connector doesn't reflect connect status atm. Sure, I will change it.

Reply:
It doesn't have to reflect Connect status. But there is a clear relationship. The Connector runs inside Connectc, so when Connect is not ready, where would you expect it to run?

Reply:
If the connector status is ready then it's very likely that the connect status is ready. If the connector status is not ready then you cannot infer anything about the connect status.

Reply:
It is more the other direction: Does it make sense to test how new connector deployes when Connect is not ready?

Reply:
Moved to phase when Kafka Connect is ready